app/controllers/idv/review_controller.rb

```diff
@@ -68,5 +76,13 @@ def submit_applicant
       idv_attempter.increment
       resolution
     end
+
+    def password
+      params.require(:user)[:password]
```

Comment:
Ok, I think I understand.

I think we want:

```ruby
params.fetch(:user, {})[:password].presence
```

The methods from strong parameters `.require` and `.permit` are only useful if we're passing entire hashes around (ex `User.create(params)`), but in this case, we just want a single key, so I think we can avoid that approach and be explicit.